.eslintrc.yaml

```diff
@@ -52,6 +52,7 @@ rules:
   no-path-concat: off
   no-proto: error
   no-redeclare: error
+  no-restricted-globals: [ error, context ]
```

Comment:
I'd love to add things like `error`, `name`, but it's probably best to keep it simple for this change.